uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1022.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1022 {
+  
+  /**
+   * Hide Fields For Contact Summary
+   */
+  public function upgrade_1022() {
+    $valuesRemoved = $this->up1022_getContactEditOptionValues();
+    $optionValues = $this->up1022_getActiveContactEditOptionValues();
```

Comment:
I know it's quite confusing with `OptionValue.value` but maybe we could make the naming a bit clearer here?

In the end we want to do a diff of "values to remove" and "active values" and only remove those that we want to that are not active, right? `$valuesRemoved` sounds like they have already been removed, and `$optionValues` is a bit vague about which option values they are.

What do you think about renaming `$valuesRemoved` to `$valuesToRemove` and `up1022_getContactEditOptionValues` to `up1022_getContactEditOptionValuesToRemove`?

And then renaming `$optionValues` to `$activeOptionValues`. We could then keep the variable `$valuesToRemove` and reassign it with the result of the diff, because really these are the "values we will remove"

Reply:
Noted thanks.